main.go

```diff
@@ -58,5 +58,15 @@ func main() {
 	l.Printf("Inventory Service, version %s starting up",
 		CreateVersionString())
 
+	db, err := NewDataStoreMongo(config.Config.GetString(SettingDb))
+	if err != nil {
+		l.Fatal("failed to connect to db")
+	}
+
+	err = db.Migrate(Version, nil)
```

Comment:
but this is the *schema* version. it goes hand in hand with the app, but only as far as it's useful for us. no reason to mix `version.go` (Commit, Tag, Branch, BuildNumber...?) into this.

 I still fail to see any connection whatsowever. The schema version changes when we say it does.